serving_with_ipex_triton/client_imagenet.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+from torchvision import transforms
+from PIL import Image
+import tritonclient.http as httpclient
+from tritonclient.utils import triton_to_np_dtype
+import os
+import time
+
+# preprocessing function
+def image_preprocess(img_path="img2.jpg"):
+    img = Image.open(img_path)
+    preprocess = transforms.Compose([
+        transforms.Resize(224),
+        transforms.CenterCrop(224),
+        transforms.ToTensor(),
+        transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+    ])
+    return preprocess(img).numpy()
+
+directory = '/home/ubuntu/ImageNet/imagenet_images'
```

Comment:
please make the path to the dataset a required parameter.

Reply:
Added mandatory parameter for dataset